actiontext/lib/action_text/system_test_helper.rb

```diff
@@ -36,6 +36,8 @@ def fill_in_rich_text_area(locator = nil, with:)
 end
 
 Capybara.add_selector :rich_text_area do
+  filter_set :_field
```

Comment:
The [`:fillable_field` selector](https://github.com/teamcapybara/capybara/blob/master/lib/capybara/selector/definition/fillable_field.rb#L21) adds itself to the [`:_field` filter set](https://github.com/teamcapybara/capybara/blob/8d32979cd9156d43d7719718e93b0bb7b9fd7fec/lib/capybara/selector.rb#L177-L221) with a collection of attributes. 

The `<trix-editor>` supports some attributes (like `[name]`, `[placeholder]`), but not others ([`[disabled]`](https://github.com/basecamp/trix/issues/410#issuecomment-309066877), `[multiple]`, `[checked]`).

Should this line call `%i[name placeholder]`?